app/js/lib/foxbox/api.js

```diff
@@ -0,0 +1,41 @@
+const p = Object.freeze({
+  settings: Symbol('settings'),
+  net: Symbol('net'),
+
+  // Private methods.
+  getURL: Symbol('getURL'),
+});
+
+export default class API {
```

Comment:
One of the ideas I'm still forming in my head is to make API work a bit like DB - it will be initialized immediately, but since all its methods are sync they will wait until user is authenticated and box is discovered....